src/views/product-tutorials-view/components/product-view-content/components/tutorials-stack/types.ts

```diff
@@ -28,5 +10,5 @@ export interface TutorialsStackProps {
   headingSlug: string
   /** Subheading to show above the tutorial cards. */
   subheading?: string
-  featuredTutorials: TutorialStackItem[]
+  featuredTutorials: ClientTutorial[]
```

Comment:
I wonder if its worth making this a smaller subtype that isn't the whole `ClientTutorial` 💭 Similar to what you were saying as formatting this data in the server or something.

Reply:
I think that'd be great! Will definitely try to aim to slip that in before June 20 - I _think_ it'll actually be quite feasible; the stubs you had already felt like a clean foundation, and now I'm realizing I think this would mostly be moving `formatCollectionCard` & `formatTutorialCard` calls into `getStaticProps` 🤔